src/ServiceControl/MessageFailures/Handlers/UnArchiveMessagesByRangeHandler.cs

```diff
@@ -50,6 +51,7 @@ public async Task Handle(UnArchiveMessagesByRange message, IMessageHandlerContex
 
             await domainEvents.Raise(new FailedMessagesUnArchived
             {
+                DocumentIds = patchedDocumentIds.Select(x => FailedMessage.GetMessageIdFromDocumentId(x.Document)).ToArray(),
```

Comment:
Is there a limit to how many of these there can be? We may want to batch them otherwise these messages get very large.

Reply:
What kind of batch size do you think we should do here?
Is there a metric we can use?